pgmpy/models/JointGaussian.py

```diff
@@ -40,3 +42,39 @@ def __init__(self, variables, mean, covariance):
         self.mean = mean
         self.covariance = covariance
         self.precision_matrix = np.linalg.inv(covariance)
+
+    def get_gradient_log_pdf(self, distribution_param, grad_log_pdf=None):
```

Comment:
@ankurankan Maybe you can just check this section, and you will get the idea of what I meant by handling grad_log_pdf in model itself.